### PR TITLE
i18n: missing strings

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-####

### CODE REVIEW

File: src/languages/_english.json [hunk 1]
```diff
@@ -1611,6 +1679,11 @@
       }
     },
     "time": {
+      "today_caps": "Today :)",
```
Comment (benisgold) [inline src/languages/_english.json:1682]: more smileys

File: src/languages/_french.json [hunk 2]
```diff
@@ -80,6 +83,14 @@
       }
     },
     "back_up": {
+      "errors": {
```
Comment (benisgold) [inline src/languages/_french.json:86]: missing smileys

File: src/languages/_french.json [hunk 3]
```diff
@@ -429,9 +441,11 @@
         "trading_at_prefix": "Trading at :)"
       },
       "search": {
-        "profiles": "Profiles :)",
-        "search_ethereum": "Search all of Ethereum :)",
-        "search_ethereum_short": "Search Ethereum :)"
+        "profiles": "Profiles",
```
Comment (benisgold) [inline src/languages/_french.json:444]: smiley!

File: src/languages/_french.json [hunk 4]
```diff
@@ -884,6 +898,19 @@
       }
     },
     "gas": {
+      "card": {
```
Comment (benisgold) [inline src/languages/_french.json:901]: :)

**Review by benisgold — CHANGES_REQUESTED** [review]
loooks good except for smileys. there's 1 spot where there's a smiley in the english translation so at least that one should be changed before merge

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js [hunk 5]
```diff
@@ -46,6 +46,8 @@ const UniqueTokenExpandedStateContent = ({
   // default to showing a loading spinner for 3D/video assets
   const [loading, setLoading] = React.useState(supports3d || supportsVideo);
 
+  console.log('asset: ', asset.animation_url);
+  console.log({ supportsVideo });
```
[the remark this answers is not in the document]
Comment (skylarbarrera) [reply src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js:50]: oops

**Review by benisgold — APPROVED** [review]

**Pull request merged**